src/main/io/serial_cli.c

```diff
@@ -968,14 +968,24 @@ static void cliShowArgumentRangeError(char *name, int min, int max)
     cliPrintf("%s must be between %d and %d\r\n", name, min, max);
 }
 
+static char *nextArg(char *currentArg)
+{
+    char *ptr = strchr(currentArg, ' ');
```

Comment:
You need to check if `ptr` returns `NULL` here.

Prefer your fix to mine, but this needs to be fixed.